client/pokemon/pokemon.ctrl.js

```diff
@@ -76,6 +76,16 @@ module.exports = class Pokemon {
   }
   parseAllProps () {
     this.parseBoxViewProps();
+    this.getRibbonClass = function(ribbon) {
+      return ribbon.replace(/[()]/g, '').toLowerCase().split(' ').join('-');
+    };
+
+    this.getRibbonName = function(ribbon) {
+      return `${ribbon.includes('Contest Memory Ribbon') ? ribbon + ' (' + this.data.contestMemoryRibbonCount + ')'
+        : ribbon.includes('Battle Memory Ribbon') ? ribbon + ' (' + this.data.battleMemoryRibbonCount + ')'
+        : ribbon}`
```

Comment:
I think the template string here isn't doing much; you have `` `${someExpression}` ``, but that could be replaced with just `someExpression`.

It might be better to use a template string for the inside bits instead:

```js
return ribbon.includes('Contest Memory Ribbon')
  ? `${ribbon} (${this.data.contestMemoryRibbonCount})`
  : ribbon.includes('Battle Memory Ribbon')
    ? `${ribbon} (${this.data.battleMemoryRibbonCount})`
    : ribbon;
```